crates/router/src/core/payment_link/payment_link_initiate/payment_link_initiator.js

```diff
@@ -56,6 +56,7 @@ function initializeSDK() {
         height: 55,
       },
     },
+    hideCardNicknameField: false,
```

Comment:
Why not use what was configured in the payment link config?

Reply:
Since displaySavedPaymentMethodsCheckbox is set to false for open payment links, it doesn't matter whether we hide the card nick name field or not. So i mapped it to it's default value in SDK, which is false.